org.deluge_torrent.deluge.yaml

```diff
@@ -0,0 +1,437 @@
+
+- name: chardet
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/ee/2d/9cdc2b527e127b4c9db64b86647d567985940ac3698eeabc7ffaccb4ea61/chardet-4.0.0.tar.gz
+    sha256: 0d6f53a15db4120f2b08c94f11e7d93d2c911ee118b6b30a04ec3ee8310179fa
+
+- name: setproctitle
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/a7/e1/e76b903af017cc4a99f823b5e4235c1136dcca6902e6b7ee4d545958ba8c/setproctitle-1.2.1.tar.gz
+    sha256: 5f0eecb27815e31799a69eb6a06b4d375d38887d079d410565b0be82da65c950
+
+- name: pyxdg
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/6f/2e/2251b5ae2f003d865beef79c8fcd517e907ed6a69f58c32403cec3eba9b2/pyxdg-0.27.tar.gz
+    sha256: 80bd93aae5ed82435f20462ea0208fb198d8eec262e831ee06ce9ddb6b91c5a5
+
+- name: MarkupSafe
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/b9/2e/64db92e53b86efccfaea71321f597fa2e1b2bd3853d8ce658568f7a13094/MarkupSafe-1.1.1.tar.gz
+    sha256: 29872e92839765e546828bb7754a68c418d927cd064fd4708fab9fe9c8bb116b
+
+- name: mako
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/72/89/402d2b4589e120ca76a6aed8fee906a0f5ae204b50e455edd36eda6e778d/Mako-1.1.3.tar.gz
+    sha256: 8195c8c1400ceb53496064314c6736719c6f25e7479cd24c77be3d9361cddc27
+
+- name: python-dbus
+  buildsystem: simple
+
+  build-commands: 
+  - python3 setup.py build
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/62/7e/d4fb56a1695fa65da0c8d3071855fa5408447b913c58c01933c2f81a269a/dbus-python-1.2.16.tar.gz
+    sha256: 11238f1d86c995d8aed2e22f04a1e3779f0d70e587caffeab4857f3c662ed5a4
+ 
+- name: deluge
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  post-install:
+  - install -Dm644 /app/share/icons/hicolor/128x128/apps/deluge.png /app/share/icons/hicolor/128x128/apps/org.deluge_torrent.deluge.png
+  - mkdir -p /app/share/applications
+  - install -Dm644 deluge/ui/data/share/applications/deluge.desktop.in /app/share/applications/org.deluge_torrent.deluge.desktop
```

Comment:
I think I prefer rename-icon and copy-icon over this.

Reply:
using rename-icon, copy-icon not required.